src/Native/Runtime/inc/eetype.h

```diff
@@ -444,6 +444,11 @@ class EEType
     bool IsGeneric()
         { return (m_usFlags & IsGenericFlag) != 0; }
 
+#if defined(CORERT)
+    ModuleManager* GetModule()
```

Comment:
Renamed to GetModuleManager. It still feels funny though that a ModuleManager (or TypeSystemManager) would have a method called GetModuleSection that doesn't require the caller to specify what module to operate on, because the Manager is the Module. We also pretty much always store the ModuleManager in a variable called module...

Maybe the naming will become clear to me as we add more functionality to the Manager.